docs/docs/rest-api/public/api/v2/types/scaling-policy.raml

```diff
@@ -0,0 +1,30 @@
+#%RAML 1.0 Library
+usage: |
+  Defines scaling policy constructs.
+types:
+  AbstractScalingPolicy:
+    discriminator: kind
+    properties:
+      kind:
+        type: string
+        enum: [ fixed ]
+  ScalingPolicyFixed:
+    type: AbstractScalingPolicy
+    discriminatorValue: fixed
+    properties:
+      instances:
+        type: integer
+        format: int32
+        description: The number of instances of this pod to start.
+        minimum: 0
+        default: 1
+      maxInstances?:
```

Comment:
The goal was to stop abusing the MARATHON_SINGLE_APP_INSTANCE label and use
a proper field

On Tue, Aug 30, 2016 at 8:36 AM, Matthias Veit <notifications@github.com>
wrote:

> In docs/docs/rest-api/public/api/v2/types/scaling-policy.raml
> <https://github.com/mesosphere/marathon/pull/4270#discussion_r76785642>:
>
> > +    discriminator: kind
> > +    properties:
> > +      kind:
> > +        type: string
> > +        enum: [ fixed ]
> > +  ScalingPolicyFixed:
> > +    type: AbstractScalingPolicy
> > +    discriminatorValue: fixed
> > +    properties:
> > +      instances:
> > +        type: integer
> > +        format: int32
> > +        description: The number of instances of this pod to start.
> > +        minimum: 0
> > +        default: 1
> > +      maxInstances?:
>
> When is this value relevant?
> Is this related to the UpgradeStrategy?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mesosphere/marathon/pull/4270/files/9c47e5011910bfc02c10bfaa43e18d9083b87436#r76785642>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACPVLG8bl_IhabLQsbEBuvh8WPA9UFbXks5qlCPlgaJpZM4Jv7F6>
> .
>